### PR TITLE
fix typo in hook placement + uninitialized hook placements

### PR DESCRIPTION
**Pull request by tom-wa**
had a typo in PRESETSTORAGE placement + forgot to initialize the globalPugins array.

### CODE REVIEW

File: src/libs/elektra/mount.c [hunk 1]
```diff
@@ -273,13 +272,14 @@ int elektraMountGlobals (KDB * kdb, KeySet * keys, KeySet * modules, Key * error
 		}
 		for (GlobalpluginPositions i = 0; i < NR_GLOBAL_PLUGINS; ++i)
 		{
+            kdb->globalPlugins[i] = NULL;
```
Comment (markus2330) [inline src/libs/elektra/mount.c]: spaces?

File: src/libs/elektra/mount.c [hunk 2]
```diff
@@ -362,11 +363,6 @@ int elektraMountModules (KDB * kdb, KeySet * modules, Key * errorKey)
 
 	while ((cur = ksNext (modules)) != 0)
 	{
-		// exclude "virtual" plugins where actual plugins could not deliever
-		// their contract properly
-		if (!strcmp (keyName (cur), "system/elektra/modules/resolver")) continue;
```
Comment (markus2330) [inline src/libs/elektra/mount.c]: here you remove really important lines that would crash every `kdb` commando accessing `system/elektra/modules`.

Comment (tom-wa) [inline src/libs/elektra/mount.c]: sorry, i really need to improve my flow of work with multiple buffers and branches ...

**Comment by markus2330** [comment]
looks much better, thank you!

**Pull request merged**